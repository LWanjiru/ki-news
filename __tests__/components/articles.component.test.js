import React from 'react';
// import expect from 'expect';
import { shallow, mount } from 'enzyme';
import Articles from './../../src/scripts/components/Articles';

it('renders without crashing', () => {
  mount(<Articles />);
});
it('renders the <ShowArticle /> component', () => {
  const wrapper = shallow(<Articles />);
  const component = wrapper.find('ShowArticle');
  expect(component.length).toBe(1);
});
it('renders articles', () => {
  const articles = [''];
  const wrapper = mount(
    <Articles articles={articles} />,
  );
  const a = wrapper.find('p');
  expect(a.text()).toBe('');
});
