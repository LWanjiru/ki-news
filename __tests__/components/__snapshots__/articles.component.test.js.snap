// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Articles component and Iframe should pass a selected value to the handleClick function 1`] = `
<div
  className="articles"
>
  <a
    href="/"
  >
    <h1
      className="heading mb-2"
    >
      <u>
        Articles
      </u>
    </h1>
  </a>
  <div
    className="iframe-btn"
  >
    <button
      aria-controls="article-frame"
      aria-expanded="false"
      className="btn btn-info active"
      onClick={[Function]}
      type="button"
    >
      Click on an article to view in Iframe
    </button>
    <p>
      <strong>
        <sup>
          *
        </sup>
        <small>
          Not all pages allow for viewing in iframe(click on linkg twice to open in new tab) 
        </small>
      </strong>
    </p>
  </div>
  <br />
  <div
    className="row"
  />
</div>
`;
